Nextcloud throws an error log when opening gui but seems to work in background

### Steps to reproduce
1. Click on nextcloud dev app

### Expected behaviour
- GUI opens

### Actual behaviour
- following message is displayed

************ CAUSE OF ERROR ************

java.lang.NoClassDefFoundError: Failed resolution of: Lokio/Buffer;
	at okhttp3.ResponseBody.create(ResponseBody.java:210)
	at okhttp3.internal.Util.<clinit>(Util.java:62)
	at okhttp3.Credentials.basic(Credentials.java:30)
	at com.owncloud.android.lib.common.OwnCloudClientFactory.createNextcloudClient(OwnCloudClientFactory.java:217)
	at com.owncloud.android.operations.RefreshFolderOperation.updatePredefinedStatus(RefreshFolderOperation.java:330)
	at com.owncloud.android.operations.RefreshFolderOperation.updateCapabilities(RefreshFolderOperation.java:304)
	at com.owncloud.android.operations.RefreshFolderOperation.updateOCVersion(RefreshFolderOperation.java:276)
	at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:229)
	at com.owncloud.android.lib.common.operations.RemoteOperation.run(RemoteOperation.java:360)
	at java.lang.Thread.run(Thread.java:919)
Caused by: java.lang.ClassNotFoundException: okio.Buffer
	... 10 more

************ APP INFORMATION ************
ID: com.nextcloud.android.beta
Version: 20201216
Build flavor: versionDev

************ DEVICE INFORMATION ************
Brand: samsung
Device: r7
Model: SM-N770F
Id: QP1A.190711.020
Product: r7naeea

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: N770FXXS6CTJ4

### Can you reproduce this problem on https://try.nextcloud.com?
- NA I believe (problem with nextcloud_dev client not server)
- Please create a test demo account and see if this still happens there.

### Environment data
Android version: see above

Device model: see above

Stock or customized system: stock

Nextcloud app version: see above

Nextcloud server version: 19.0.2

Reverse proxy: no

### Logs
#### Web server error log
Server does not seem involved

Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

Same issue here. Tried multiple servers with and without 2FA, login via App Token and directly.

```
************ CAUSE OF ERROR ************

java.lang.NoClassDefFoundError: Failed resolution of: Lokio/Buffer;
 at okhttp3.ResponseBody.create(ResponseBody.java:210)
 at okhttp3.internal.Util.<clinit>(Util.java:62)
 at okhttp3.Credentials.basic(Credentials.java:30)
 at com.owncloud.android.lib.common.OwnCloudClientFactory.createNextcloudClient(OwnCloudClientFactory.java:217)
 at com.owncloud.android.operations.RefreshFolderOperation.updatePredefinedStatus(RefreshFolderOperation.java:330)
 at com.owncloud.android.operations.RefreshFolderOperation.updateCapabilities(RefreshFolderOperation.java:304)
 at com.owncloud.android.operations.RefreshFolderOperation.updateOCVersion(RefreshFolderOperation.java:276)
 at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:229)
 at com.owncloud.android.lib.common.operations.RemoteOperation.run(RemoteOperation.java:360)
 at java.lang.Thread.run(Thread.java:929)
Caused by: java.lang.ClassNotFoundException: okio.Buffer
 ... 10 more

************ APP INFORMATION ************
ID: com.nextcloud.android.beta
Version: 20201216
Build flavor: versionDev

************ DEVICE INFORMATION ************
Brand: HUAWEI
Device: HWVOG
Model: VOG-L29
Id: HUAWEIVOG-L29
Product: VOG-L29EEA

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: 10.1.0.150C431
```

--- a/build.gradle
+++ b/build.gradle
@@ -13,9 +13,6 @@ buildscript {
         google()
         jcenter()
         maven {
-            url 'https://oss.sonatype.org/content/repositories/snapshots/'
-        }
-        maven {
             url 'https://plugins.gradle.org/m2/'
         }
         mavenCentral()
@@ -77,7 +74,6 @@ repositories {
     google()
     jcenter()
     maven { url "https://jitpack.io" }
-    maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
 
     flatDir {
         dirs 'libs'

